services/community-site/src/interfaces/Validator.ts

```diff
@@ -15,6 +15,12 @@ export interface ContractValidator {
   info: ContractValidatorInfo;
 }
 
+export enum ValidatorStatus {
+  Grey = 'grey',
+  Yellow = 'yellow',
```

Comment:
Also don't like representing colors in an interface. Maybe we could call them:
- ELIGIBLE
- ELIGIBLE_INACTIVE
- NOT_ELIGIBLE